### PR TITLE
EPUB accessibility techniques ordering schema

### PR DESCRIPTION
**Pull request by clapierre**
Adjusted the ordering of the techniques based on our call 3/7/24.
updated to using EPUB accessibility metadata instead of schema.org
Fixed some outdated URLs to older 3.2 / 1.0 specs.

### CODE REVIEW

**Comment by clapierre** [comment]
Can we merge this?  I have the next set of edits to push but would like this approved first :)

File: UX-Guide-Metadata/draft/techniques/epub-metadata/index.html [hunk 1]
```diff
@@ -70,23 +70,23 @@ <h2>EPUB Accessibility Metadata</h2>
...
+				    <aside class="example" id="epub-a11y-metadata-example-1">
                         <pre><code class="xpath">TBD</code></pre>
                     </aside>
 				</section>
 				<section>
-				    <h4 id="schema-org-describing-an-full-audio-book">Schema.org metadata describing an audiobook</h4>
-				        <p>Here is an example of an Schema.org OPF Metadata for describing an audiobook, which will be used as a reference point for the some of the following examples on EPUB accessibility metadata: the results of the XPath shown are based on this example.</p>
-				        <aside class="example" id="schema-org-example-2">
+				    <h4 id="epub-a11y-metadata-describing-an-full-audio-book">EPUB accessibility metadata describing an audiobook</h4>
+				        <p>Here is an example of an accessibility OPF Metadata for describing an audiobook, which will be used as a reference point for the some of the following examples on EPUB accessibility metadata: the results of the XPath shown are based on this example.</p>
```
Comment (gautierchomel) [inline UX-Guide-Metadata/draft/techniques/epub-metadata/index.html:88]: ... for some of... (There's an additional "the" to suppress)

**Review by gautierchomel — APPROVED** [review]

**Comment by avneeshsingh** [comment]
Please go ahead and merge.

File: UX-Guide-Metadata/draft/techniques/epub-metadata/index.html (hunk 1 above)
Comment (clapierre) [reply UX-Guide-Metadata/draft/techniques/epub-metadata/index.html]: Fixed thanks Gautier

**Pull request merged**